definitions/procedures/backup/prepare_directory.rb

```diff
@@ -6,18 +6,29 @@ class PrepareDirectory < ForemanMaintain::Procedure
       param :backup_dir, 'Directory where to backup to', :required => true
       param :preserve_dir, 'Directory where to backup to', :flag => true
       param :incremental_dir, 'Changes since specified backup only'
+      param :online_backup, 'Select for online backup', :flag => true, :default => false
     end
 
+    # rubocop:disable Metrics/MethodLength
     def run
       puts "Creating backup folder #{@backup_dir}"
 
       unless @preserve_dir
```

Comment:
I think this would actually make a lot more sense, but I can understand why you don't want to touch those too much
```suggestion
      if @preserve_dir
        FileUtils.rm(Dir.glob(File.join(@backup_dir, '.*.snar')))
      else
        puts "Creating backup folder #{@backup_dir}"
```

Reply:
(moved the puts, but not the `rm`)

I think what my brain tries to fight is the fact that "not preserve_dir" is the *primary* mode of operation of this code, so it should come *first* vs the else branch that is, well, "else" ;-)